pkgs/os-specific/linux/firmware/system76-firmware/default.nix

```diff
@@ -29,11 +29,11 @@ rustPlatform.buildRustPackage rec {
     done
   '';
 
-  meta = {
+  meta = with lib; {
     description = "Tools for managing firmware updates for system76 devices";
     homepage = "https://github.com/pop-os/system76-firmware";
-    license = lib.licenses.gpl3;
-    maintainers = [ lib.maintainers.shlevy ];
-    platforms = lib.platforms.linux;
+    license = licenses.gpl3;
```

Comment:
gpl3Only or gpl3Plus?

Reply:
Not sure. Don't have much experience with licenses. Here it is:
https://github.com/pop-os/system76-firmware/blob/master/LICENSE
What do you think?

Reply:
It does not mention later anywhere so it is gpl3Only.

Reply:
I have changed license to gpl3Only.